src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,111 @@
+package duke;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.exception.WriteToStorageException;
+import duke.task.Task;
+
+
+/**
+ * Represents a list of Task objects. Contains functions to manipulate tasklist.
+ */
+public class TaskList {
+    private Storage store;
+    private List<Task> tasks;
+
+
+    /**
+     * To instantiate a TaskList, pass in Storage object.
+     *
+     * @param store  Storage Object to store Task data.
+     */
+    public TaskList(Storage store) {
+        this.store = store;
+        this.tasks = new ArrayList<>();
+        store.initialiseTasks(tasks);
+    }
```

Comment:
Might be better to phrase `To instantiate...` as `Public constructor for TaskList object that requires a Storage object` or something along those lines.